### ISSUE TITLE
studio.ml domain is down

### ISSUE THREAD

**Issue by jax79sg** [studioml/studio#352]
Hi, can't access your main domain where your documentation is linked to.

**Comment by jax79sg** [comment]
Hi thanks for the reply. The hyperlinks in the link you provided are linking to studio.ml urls, which are broken. The temp solution is to provide the head here. https://github.com/studioml/studio/tree/master/docs

**Comment by karlmutch** [comment]
FYI
I am still waiting on a reply related to the account being used on studioml.readthedocs.org.

**Comment by karlmutch** [comment]
Closed due to inactivity.  If this should occur again please open a new ticket.  Thanks

**Comment by karlmutch** [comment]
can you post any more details such as messages or logs. Works for a quick test on our side.

**Comment by sotte** [comment]
Now it works. Weird.

Using Firefox 64 I got a warning that the ssl certificate was not valid.
